packages/@aws-cdk/aws-glue/test/connection.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from '@aws-cdk/assert';
+import { Stack } from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import * as glue from '../lib';
+import * as ec2 from '@aws-cdk/aws-ec2';
+
+export = {
```

Comment:
Uhm. Maybe I did a mental shortcut saying "that's all you needed". Jest test look a tiny bit different than Nodeunit tests. So, this file would be something like:

```ts
describe('Glue Connection', () => {
  test('can be created without a VPC', () => {
    // ...
  });

  test('can be created with a VPC', () => {
    // ...
  });
});
```